site/pages/home/contact-us-slice/test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import ContactUs from '.';
+import Form from './form/';
+import Success from './success/';
+
+describe('Contact Us Slice', () => {
+  it('shows pre-signup content if the form has not been submitted yet - default state', () => {
+    const wrapper = shallow(<ContactUs />);
+    expect(wrapper.find(Form)).to.have.length(1);
+  });
+
+  it('shows success content if the form has been submitted successfully', () => {
+    const wrapper = shallow(<ContactUs />);
+    wrapper.setState({
+      success: true,
+    });
+    expect(wrapper.find(Success)).to.have.length(1);
+  });
+});
```

Comment:
Would be great to have the logic tested as well as the rendering. I'm interested in the logic around hitting the API. :)

Also we can test the rendering with black-box testing by passing in the API client, instead of altering the private state of the component.